### PR TITLE
1.1.0 docs update

### PR DESCRIPTION
**Pull request by yadudoc**
# Description

Changelog for `v1.1.0` release

## Type of change

Choose which options apply, and delete the ones which do not apply.

- Documentation update

### CODE REVIEW

File: docs/devguide/changelog.rst [hunk 1]
```diff
@@ -2,6 +2,62 @@ Changelog
 =========
 
 
+Parsl 1.1.0
+-----------
+
+Released on April 15th, 2021
```
Comment (benclifford) [inline docs/devguide/changelog.rst:8]: wrong

**Review by benclifford — DISMISSED** [review]

File: docs/devguide/changelog.rst [hunk 2]
```diff
@@ -2,6 +2,62 @@ Changelog
...
+Kelly L. Rowland @kellyrowland, Lars Bilke @bilke, Logan Ward @WardLT, Kirill Nagaitsev @Loonride,
+Marcus Schwarting @meschw04, Matt Baughman @mattebaughman, Mihael Hategan @hategan, @radiantone,
+Rohan Kumar @rohankumar42, Sohit Miglani @sohitmiglani, Tim Shaffer @trshaffer,
+Tyler J. Skluzacek @tskluzac, Yadu Nand Babuji @yadudoc, and Zhuozhao Li @ZhuozhaoLi
+
+Deprecated and Removed features
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Python 3.5 is no longer supported.
+* Remove almost definitely broken Jetstream provider (#1821)
```
Comment (danielskatz) [inline docs/devguide/changelog.rst:23]:
```suggestion
* Almost definitely broken Jetstream provider removed (#1821)
```

File: docs/devguide/changelog.rst [hunk 3]
```diff
@@ -2,6 +2,62 @@ Changelog
...
+
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+* Allow HTEX to set CPU affinity (#1853)
+
+* New serialization system to replace IPP serialization (#1806)
+
+* Support python 3.9
```
Comment (danielskatz) [inline docs/devguide/changelog.rst:32]:
```suggestion
* Support for Python 3.9
```

File: docs/devguide/changelog.rst [hunk 4]
```diff
@@ -2,6 +2,62 @@ Changelog
...
+
+* @join_apps are a variation of @python_apps which an app to launch
```
Comment (danielskatz) [inline docs/devguide/changelog.rst:34]:
```suggestion
* @join_apps are a variation of @python_apps where an app can launch
```

**Review by danielskatz — DISMISSED** [review]
good other than 3 small wording changes

File: docs/devguide/changelog.rst [hunk 5]
```diff
@@ -2,6 +2,62 @@ Changelog
...
+  completed.
+    
+  These are described more fully in docs/userguide/joins.rst
+
+* Monitoring:
+    hub.log is now named monitoring_router.log
+    Remove denormalised workflow duration from monitoring db (#1774)
+    Remove hostname from status table (#1847)
+    Clarify distinction between tasks and tries to run tasks (#1808)
+    Replace `done` state with exec_done and memo_done (#1848)
```
Comment (ZhuozhaoLi) [inline docs/devguide/changelog.rst:45]: add quotes around? `exec_done` and `memo_done`

File: docs/devguide/changelog.rst [hunk 6]
```diff
@@ -2,6 +2,62 @@ Changelog
...
+  completed.
+
+  These are described more fully in docs/userguide/joins.rst
+
+* Monitoring:
+    hub.log is now named monitoring_router.log
+    Remove denormalised workflow duration from monitoring db (#1774)
+    Remove hostname from status table (#1847)
+    Clarify distinction between tasks and tries to run tasks (#1808)
+    Replace `done` state with exec_done and memo_done (#1848)
```
Comment (ZhuozhaoLi) [inline docs/devguide/changelog.rst:45]:
```suggestion
    Replace `done` state with `exec_done` and `memo_done` (#1848)
```

File: docs/devguide/changelog.rst (hunk 1 above)
Comment (danielskatz) [reply docs/devguide/changelog.rst]: addressed

**Review by benclifford — APPROVED** [review]

**Pull request merged**